ranker/show/show.html

```diff
@@ -0,0 +1,53 @@
+	<body>
+		<div id="wrapper">
+			<div id="head">
+				<div id="actions">
+					<ul class="normalRes">
+						
+						
+					</ul>
+
+				</div>
+			</div>
+
+			<div style="height: 637px;" id="tabs">
+				
+					<div class="tCont result active" id="result"><iframe style="height: 639px;" sandbox="allow-forms allow-popups allow-scripts allow-same-origin" src="show_files/ab.html"></iframe></div>
+          
+												
+				
+			</div>
+		</div>
+
+	  
+	    <script src="show_files/ga.js" type="text/javascript"></script>
+	    	
+
+</body></html>
```

Comment:
Rename directories to variables that reflect distinction between frontend and backend of the application.